src/renderer/modals/FullNode/helpers.js

```diff
@@ -0,0 +1,55 @@
+// @flow
+import { ipcRenderer } from "electron";
+import resolveUserDataDirectory from "~/helpers/resolveUserDataDirectory";
+import { setEnvOnAllThreads } from "~/helpers/env";
+import {
+  editSatsStackConfig,
+  stringifySatsStackConfig,
+  parseSatsStackConfig,
+} from "@ledgerhq/live-common/lib/families/bitcoin/satstack";
+import type { SatsStackConfig } from "@ledgerhq/live-common/lib/families/bitcoin/satstack";
+import path from "path";
+const lssFileName = "lss.json";
+
+export const saveLSS = async (lssConfig: SatsStackConfig) => {
+  const userDataDirectory = resolveUserDataDirectory();
+  const filePath = path.resolve(userDataDirectory, lssFileName);
+
+  if (filePath) {
+    const configStub = {
+      node: { host: "", username: "", password: "" },
+      accounts: [],
+    };
+    const maybeExistingConfig = (await loadLSS()) || configStub;
+    const updated = editSatsStackConfig(maybeExistingConfig || { accounts: [] }, lssConfig);
+    await ipcRenderer.invoke("generate-lss-config", filePath, stringifySatsStackConfig(updated));
```

Comment:
with the current logic, this could fail because you have silented the error case of writing the file.
either don't try {} catch {} on the other side OR do a condition here.

I tend to think we must not try catch on other side because you would need to display an error on the UI, right?

Reply:
Not try/catch on the main thread anymore, all errors will be thrown, let's see how that goes